libkbfs/folder_branch_ops.go

```diff
@@ -1840,13 +1843,26 @@ func (fbo *folderBranchOps) finalizeMDWriteLocked(ctx context.Context,
 			fbo.log.CDebugf(ctx, "Conflict: %v", err)
 			mergedRev = md.Revision
 
+			if excl == WithEXCL {
```

Comment:
I know it's a pain, but I think we should consider moving all the `EXCL` constants to say `Excl` instead.  To me it really stands out, since it's not an acronym and none of our other consts use the all caps convention (which is usually frowned-upon in golang).

I know we're borrowing it from POSIX, but since this isn't exposed anywhere that other POSIX flags are exposed, I think it makes the most sense to change everything to `Excl` instead.  What do you think?